docs/topics/mta-rn-technical-changes-7-0-0.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes-7.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-technical-changes-7-0-0_{context}"]
```

Comment:
[dylan](https://github.com/dymurray) && [shveta](https://github.com/sshveta)
Please can I ask for you input on any technical changes you think should be listed